volttron/platform/vip/pubsubwrapper.py

```diff
@@ -179,4 +211,4 @@ def _peer_unsubscribe(self, prefix, bus=''):
                 subscribers.discard(peer)
                 if not subscribers:
                     del subscriptions[prefix]
-        self.vip.pubsub.unsubscribe(peer, prefix, self._collector, bus=bus)
+                    self.vip.pubsub.unsubscribe(peer, prefix, self._collector, bus=bus)
```

Comment:
I think we also need to do this up up on line 207 after 

del subscriptions[topic]